src/app/loginComponents/accounts/external/accounts.component.html

```diff
@@ -69,6 +69,9 @@ <h3>
         {{ row.message }}
         <span *ngIf="row.isLinked">If your token has expired, relink your account.</span>
       </div>
+      <div *ngIf="row.source === TokenSource.ORCID">
+        See <a target="_blank" rel="noopener noreferrer" href="https://info.orcid.org/what-is-orcid/">ORCID</a> and <a target="_blank" rel="noopener noreferrer" href="https://docs.dockstore.org/en/develop/end-user-topics/ORCID.html">docs</a> for information on what is ORCID.
```

Comment:
Should use `Dockstore.DOCUMENTATION_URL`, especially since we're moving away from `develop`.